eng/testing/runtimeConfiguration.targets

```diff
@@ -1,45 +1,31 @@
 <Project> 
   <PropertyGroup>
-    <!-- Copies the app.config file to the OutDir. -->
-    <TestRuntimeConfigurationFile Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">$(MSBuildThisFileDirectory)netfx.exe.config</TestRuntimeConfigurationFile>
-
     <!-- By default copy the test runtime config file for executable test projects (+ test support projects). -->
-    <GenerateRuntimeConfigurationFiles Condition="'$(IsTestProject)' == 'true' or '$(OutputType.ToLower())' == 'exe'">true</GenerateRuntimeConfigurationFiles>
+    <IsTestProjectOrExecutable Condition="'$(IsTestProject)' == 'true' or
+                                          '$(OutputType.ToLower())' == 'exe'">true</IsTestProjectOrExecutable>
+    <GenerateRuntimeConfigurationFiles Condition="'$(GenerateRuntimeConfigurationFiles)' == '' and
+                                                  '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
+                                                  '$(IsTestProjectOrExecutable)' == 'true'">true</GenerateRuntimeConfigurationFiles>
+    <GenerateAppConfigurationFile Condition="'$(IsTestProjectOrExecutable)' == 'true' and
+                                             '$(TargetFrameworkIdentifier)' == '.NETFramework'">true</GenerateAppConfigurationFile>
+    <AppConfig Condition="'$(GenerateAppConfigurationFile)' == 'true'">$(MSBuildThisFileDirectory)netfx.exe.config</AppConfig>
+    <!-- TODO: Change in RemoteExecutor -->
```

Comment:
Open an issue?

Reply:
No need for that, I won't forget and will submit the change soon after this PR is in. The extra work for having a tracking issue isn't worth it.